index.html

```diff
@@ -13424,193 +13424,346 @@ <h2>IANA Considerations</h2>
+          directed graphs, the serialization SHOULD NOT be passed through a
+          code execution mechanism such as JavaScript's <code>eval()</code>
+          function to be parsed. An (invalid) document may contain code that,
+          when executed, could lead to unexpected side effects compromising
+          the security of a system.</p>
+        <p>When processing JSON-LD documents, links to remote contexts and frames are
+          typically followed automatically, resulting in the transfer of files
+          without the explicit request of the user for each one. If remote
+          contexts are served by third parties, it may allow them to gather
+          usage patterns or similar information leading to privacy concerns.
+          Specific implementations, such as the API defined in the
+          JSON-LD 1.1 Processing Algorithms and API specification [[JSON-LD11-API]],
+          may provide fine-grained mechanisms to control this behavior.</p>
+        <p>JSON-LD contexts that are loaded from the Web over non-secure connections,
+          such as HTTP, run the risk of being altered by an attacker such that
+          they may modify the JSON-LD <a>active context</a> in a way that
+          could compromise security. It is advised that any application that
+          depends on a remote context for mission critical purposes vet and
+          cache the remote context before allowing the system to use it.</p>
+        <p>Given that JSON-LD allows the substitution of long IRIs with short terms,
+          JSON-LD documents may expand considerably when processed and, in the worst case,
+          the resulting data might consume all of the recipient's resources. Applications
+          should treat any data with due skepticism.</p>
+        <p>As JSON-LD places no limits on the IRI schemes that may be used,
+          and vocabulary-relative IRIs use string concatenation rather than
+          IRI resolution, it is possible to construct IRIs that may be
+          used maliciously, if dereferenced.</p>
+      </dd>
+      <dt>Interoperability considerations:</dt>
+      <dd>Not Applicable</dd>
+      <dt>Published specification:</dt>
+      <dd>http://www.w3.org/TR/json-ld</dd>
+      <dt>Applications that use this media type:</dt>
+      <dd>Any programming environment that requires the exchange of
+        directed graphs. Implementations of JSON-LD have been created for
+        JavaScript, Python, Ruby, PHP, and C++.
+      </dd>
+      <dt>Additional information:</dt>
+      <dd>
+        <dl>
+          <dt>Magic number(s):</dt>
+          <dd>Not Applicable</dd>
+          <dt>File extension(s):</dt>
+          <dd>.jsonld</dd>
+          <dt>Macintosh file type code(s):</dt>
+          <dd>TEXT</dd>
+        </dl>
+      </dd>
+      <dt>Person &amp; email address to contact for further information:</dt>
+      <dd>Ivan Herman &lt;ivan@w3.org&gt;</dd>
```

Comment:
Contact should probably be @pchampin, who is the current WG Team Contact. But, I wonder if W3C has a policy for this, as team members also come and go over time.

Reply:
@pchampin, either you should be the contact, or someone more permanent at W3C.

```suggestion
      <dd>Pierre-Antoine Champin &lt;pierre-antoine@w3.org&gt;</dd>
```